### PR TITLE
Configuration option for number of roundstart mice

### PR DESCRIPTION
**Pull request by coiax**
This was actually requested, because it turns out testing powernets is
hard if mice randomly chew wires sometimes.

### CODE REVIEW

File: code/controllers/configuration.dm [hunk 1]
```diff
@@ -258,6 +258,8 @@
 	var/arrivals_shuttle_dock_window = 55	//Time from when a player late joins on the arrivals shuttle to when the shuttle docks on the station
 	var/arrivals_shuttle_require_safe_latejoin = FALSE	//Require the arrivals shuttle to be operational in order for latejoiners to join
 
+	var/mice_roundstart = 10 // how many wire chewing rodents spawn at roundstart.
```
Comment (Jalleo) [inline code/controllers/configuration.dm]: I suggest we make this 20 just because the Janitor needs to actually put traps down.

**Review by Jalleo — APPROVED** [review]

File: code/controllers/configuration.dm (hunk 1 above)
Comment (MrStonedOne) [inline code/controllers/configuration.dm]: No, it has to be 10, this adds a new config option so it's suppose to preserve old behavior. Normally its 10, changing it to 20 would have to be in another pr

**Comment by AnturK** [comment]
Stop with the fucking configs for every single thing. If anything this should be map property.

**Comment by PKPenguin321** [comment]
Literally when will anybody feel the need to touch the mouse config

**Comment by MrStonedOne** [comment]
>not reading the fucking PR op.

Like really pk, you should fucking know better

**Comment by PKPenguin321** [comment]
Oh, I misinterpreted it
RIP the 10 mice that all spawn on the same tile on runtimestation

**Comment by duncathan** [comment]
this is fine

**Comment by duncathan** [comment]
@coiax conflicts tho

**Comment by optimumtact** [comment]
this is retarded, if your'e testing powernets just remove the mice chewing behaviour.

then go ahead and pr that to the repo as well

**Comment by coiax** [comment]
@duncathan citrus has opinions.

**Comment by coiax** [comment]
Asking mappers to modify code to test their maps seems... silly.

**Comment by xxalpha** [comment]
If you want to test powernets you don't need to push this code into the master branch, keep it in your test build.

**Comment by coiax** [comment]
I'm not the one testing powernets.

**Comment by PKPenguin321** [comment]
wait why not just del-all mouse

**Comment by MrStonedOne** [comment]
There is a mapping verb category for debug verbs. would be a good idea to put it there

**Comment by duncathan** [comment]
this is good. nice chewing wires suckssssss for small downstreams. I will use this config option
